src/snippets/get-started/node-js-hono/Step2.mdx

```diff
@@ -0,0 +1,14 @@
+import Step2SetEnv from "@/snippets/rate-limiting/quick-start/shared/Step2SetEnv.mdx";
+
+<Step2SetEnv />
+Next you need to update the dev command in your `package.json` to use the
+`.env.local` file.
```

Comment:
Do we want the warning note for dotenv like we have elsewhere?

Reply:
Yeah good suggestion, I'll add one.